docs/contribute/style_guide.rst

```diff
@@ -61,7 +64,7 @@ File and Folder Names
 
```

Comment:
`**C++**` -> 
```
File and Folder Names in C++
~~~~~~~~~~~~~~~~~~~~~~~~
```

`**Python**` -> analogously

So that the headlines are analogous to the "Python Type Annotations" headline.